tests/ipython/testdisplayhooks.py

```diff
@@ -73,3 +73,47 @@ def test_store_render_combined(self):
                       'application/vnd.holoviews_exec.v0+json',
                       'image/svg+xml', 'image/png'}
         self.assertEqual(set(data), mime_types)
+
+
+class TestBokehTheming(TestDisplayHooks):
```

Comment:
Please move the tests inside ``tests/plotting/bokeh``.

Reply:
Fixed in 61cb658